<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedNetworkSetup</class>
 <widget class="QWidget" name="AdvancedNetworkSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: #17394A;
background: rgb(240, 240, 240);
</string>
  </property>
  <widget class="QLabel" name="networkoptionsdescriptionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
   <property name="text">
    <string>configure advanced networking options</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="networkoptionsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
   <property name="text">
    <string>networking</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="networkoptionsnextButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>65</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color:#17394A;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 rgb(240,240,240), stop: 1 rgb(210,210,210));
border-radius: 11px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 rgb(240,240,240), stop: 1 rgb(180,180,180));
}

QPushButton:pressed {
background: qradialgradient(cx: 0.1, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 rgb(240,240,240), stop: 1 rgb(180,180,180));
}


</string>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gwlineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="ipaddressLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
   <property name="text">
    <string>ip:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="maskLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
   <property name="text">
    <string>mask:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="dns2lineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="masklineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="primarydnsLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
   <property name="text">
    <string>dns1:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="gatewayLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
   <property name="text">
    <string>gw:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="secondaryDNSLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
   <property name="text">
    <string>dns2:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipaddresslineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dns1lineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #17394A;
background: rgb(240, 240, 240, 0);
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
